docs/source/usage.rst

```diff
@@ -28,6 +28,64 @@ Create executable jar
 
 This command will create an executable jar in your current working directory under /target.
 
+Import variants
+---------------
+
+Variants can be imported to the store by using a `POST` request to the corresponding **/variants** endpoint.
+The variants have to be associated with metadata and the following properties have to be specified in `JSON` schema.
+
+.. code-block:: javascript
+
+    "case": {"identifier": "string"},
+    "variant_annotation": {"version": "string", "name": "string", "doi": "string"},
+    "variant_calling": {"version": "string", "name": "string", "doi": "string"},
+    "reference_genome": {"source": "string", "version": "string", "build": "string"},
+    "is_somatic": "boolean",
+    "samples": [{"identifier": "string", "cancerEntity": "string"}]
+
+For example using `curl` your upload command would look like this:
+
+.. code-block:: bash
+
+    curl -X 'POST' \
+    '${host}:${variantstore-port}/variants' \
+    -H 'accept: application/json' \
+    -H 'Content-Type: multipart/form-data' \
+    -F metadata='{"case": {"identifier": "do1234"}, "variant_annotation": {"version": "bioconda::4.3.1t", "name": "snpeff", "doi": "10.4161/fly.19695"}, "is_somatic": "true", "samples": [{"identifier": "S123456", "cancerEntity": "HCC"}], "reference_genome": {"source": "GATK", "version": "unknown", "build": "hg38"}, "variant_calling": {"version": "bioconda::2.9.10", "name": "Strelka", "doi": "10.1038/s41592-018-0051-x"}}' \
+    -F files=@/path/to/variants.vcf.gz
+
+Import additional gene information
+----------------------------------
+
+Information on genes, such as `biotype`, `name`, and `description`, can be imported to the store in `gff3` format.  
+
+For example using `curl` your upload command would look like this:
+
+.. code-block:: bash
+
+    curl -X 'POST' \
+    '${host}:${variantstore-port}'/genes' \
+    -H 'accept: application/json' \
+    -H 'Content-Type: multipart/form-data' \
+    -F 'files=@/path/to/genes.GRCh38.87.gff3'
+
+This feature is currently supported for `gff3` files dervied from Ensembl. The Ensembl version (87 in the example above) is expected to be part of the file name otherwise the corresponding database field will be empty.
```

Comment:
```suggestion
This feature is currently supported for `gff3` files derived from Ensembl. The Ensembl version (87 in the example above) is expected to be part of the file name otherwise the corresponding database field will be empty.
```

Reply:
Good catch 👍